tests/ert3/engine/integration/test_engine.py

```diff
@@ -5,7 +5,7 @@
 
 import ert3
 
-from tests.ert3.conftest import (
+from .misc import (
```

Comment:
Does the `from .misc` differ from doing just `from misc` ?

Reply:
. (relative import) implies that the current module has a parent module,
.<module> is no longer used

Reply:
And do we need the added init in the folder? Or can we just drop the init and the .

Reply:
all inits have been dropped